Formula/libcdio-paranoia.rb

```diff
@@ -0,0 +1,20 @@
+class LibcdioParanoia < Formula
+  desc "The cd-paranoia audio-cd ripper based on libcdio"
```

Comment:
`Audio CD ripper based on libcdio` would be shorter

Reply:
Changed the description.